deployment/media-insights-dataplane-streaming-stack.template

```diff
@@ -25,10 +25,13 @@ Resources:
   AnalyticsStream:
     Type: "AWS::Kinesis::Stream"
     Properties:
-        ShardCount: !Ref KinesisShardCount
-        StreamEncryption:
-          EncryptionType: "KMS"
-          KeyId: "alias/aws/kinesis"
+      ShardCount: !Ref KinesisShardCount
+      StreamEncryption:
+        EncryptionType: "KMS"
+        KeyId: "alias/aws/kinesis"
+      Tags:
+        - Key: "environment"
+          Value: "mie"
```

Comment:
Question: Is there any advantage/disadvantage for using a static tag vs. something generated by the stack like the SHORT_UUID?

Seems like the tag space between 2 stacks probably won't collide, so I don't think this impacts multiple deployments.

Reply:
I'm not sure what purpose a UUID would provide in this case. This PR relates to #292, and I thought that issue was intended to address the need for users to be able to easily identify MIE related resources in AWS Cost Management. If users need to filter based on a particular stack, then they can use the aws:cloudformation:stack-id tag, along with the user-defined environment tag that we're introducing in this PR.

Reply:
Well, I think I was confusing this with the other tag related issue we have open (#328).

However, I'm also thinking about the case when there are multiple MIE stacks deployed and you want to distinguish between costs for each stack.